util/src/main/java/org/kaazing/gateway/util/GL.java

```diff
@@ -20,6 +20,8 @@
 
 public class GL {
 
+    public static final String CLUSTER_LOGGER_NAME = "ha";
```

Comment:
But all the log statements are already using the GL logger ?   How is a static constant located in this class any different than in other class ?